packages/amplify-cli-core/src/state-manager/pathManager.ts

```diff
@@ -89,6 +94,12 @@ export class PathManager {
       PathConstants.BackendConfigFileName,
     ]);
 
+  getDotAWSDirPath = (): string => path.normalize(path.join(homedir(), PathConstants.DotAwsDir));
+
+  getCredentialsFilePath = (): string => path.normalize(path.join(this.getDotAWSDirPath(), PathConstants.Credentials));
```

Comment:
can we call these `getAWS*` methods as `getConfigFilePath` is way to generic even for the CLI.